sympy/solvers/ode/tests/test_ode.py

```diff
@@ -33,6 +33,152 @@
 # dsolve() to return different results on different machines)
 
 
+def _ode_solver_test(ode_examples,our_hint='default'):
+    for example in ode_examples['examples']:
+        eq = ode_examples['examples'][example]['eq']
+        sol = ode_examples['examples'][example]['sol']
+        if our_hint not in classify_ode(eq):
+            message = """
+                        Hint did not match the example {example}
```

Comment:
Have you tested to see what this prints out when there is a test failure?

Reply:
yes, I have tested them.

Reply:
The output is unnecessarily indented:
```
AssertionError: 
                        Different solution found from dsolve for example euler_hom_01

                        The ODE is:
                        Eq(2*x**2*Derivative(f(x), (x, 2)) - 3*x*Derivative(f(x), x), 0)

                        The expected solution was
                        [Eq(f(x), C2*x**(5/2) + C2)]

                        What dsolve returned is
                        Eq(f(x), C1 + C2*x**(5/2))
```

Reply:
One level of indentation might be useful but this is too much

Reply:
AssertionError: Different solution found from dsolve for example euler_hom_01
    
                    The ODE is:
                    Eq(2*x**2*Derivative(f(x), (x, 2)) - 3*x*Derivative(f(x), x), 0)
    
                    The expected solution was
                    [Eq(f(x), C1 + C2*x**5)]
    
                    What dsolve returned is
                    Eq(f(x), C1 + C2*x**(5/2))
does this indentation look good?

Reply:
It doesn't really need to be indented at all. I would just make it not indented.

Reply:
sorry I didn't get you?

Reply:
The output is only indented because you have indented the triple-quoted string. We don't actually want it to be indented

Reply:
so can you explain more what change are you asking to do?